redis-snapshot-replicator/README.md

```diff
@@ -0,0 +1,23 @@
+TODO manual
+
+https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/backups-exporting.html#backups-exporting-grant-access
+
+Choose Permissions.
+
+Choose Access Control List.
+
+Under Access for other AWS accounts, choose + Add account.
+
+All other regions:
+
+540804c33a284a299d2547575ce1010f2312ef3da9b3a053c8bc45bf233e4353
```

Comment:
I'm assuming this will get some extra description?

Reply:
I've updated the README file

Reply:
I don't really understand what this string represents